packages/grid/_modules_/grid/hooks/features/keyboard/useGridKeyboard.ts

```diff
@@ -53,9 +53,8 @@ export const useGridKeyboard = (apiRef: GridApiRef): void => {
 
   const handleCellKeyDown = React.useCallback(
     (params: GridCellParams, event: React.KeyboardEvent) => {
-      // The target is not an element when triggered by a Select inside the cell
-      // See https://github.com/mui-org/material-ui/issues/10534
-      if ((event.target as any).nodeType === 1 && !isGridCellRoot(event.target as Element)) {
+      // Ignore portal
```

Comment:
It is not really just about blocking portal. For example, keyboard navigation on ActionCell is blocked because of this condition
```suggestion
      // Do not apply shortcuts if the focus is not on the cell root component
```

Reply:
The implementation of `isGridCellRoot` is wrong. It's not considering that the element might not be an immediate child of the parent. That's why the keyboard navigation doesn't work in the actions cell. I'll fix this in another PR.